### PR TITLE
chore: Adds new SU4 flight model parameters

### PR DESCRIPTION
**Pull request by donstim**
Fixes #[issue_no]

## Summary of Changes

Adds new flight model parameters added by Sim Update 4 to the flight_model.cfg file. Of these, only 2 of the new parameters are used at this time:
`modern_fm_only = 1` 1 (true) forces use of modern flight model regardless of what user selected in MSFS options menu. 0 (false) allows use of user-selected flight model. **(This was already added by #4930, so this is not a change) **

`air_spoiler_limit` Spoiler max deflection angle when on the ground (absolute value) (DEGREES)
(NOTE: the definition of the current parameter `spoiler_limit' has changed to Spoiler max deflection angle when in the air (absolute value) (DEGREES)

The other new parameters are not being used currently, but are provided for possible future use. They have been commented out in this flight_model.cfg file.

Also, the value of the `fuel_dump_rate` parameter has been set to zero since the A320 series does not have fuel dump capability.

## Additional context

Discord username (if different from GitHub): donbikes#4084

## Testing instructions

Verify that in-flight and on ground spoiler deflection angles have not been affected by this PR. There are no other active changes made by this PR, so there should not be any changes to any airplane behavior.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg [hunk 1]
```diff
@@ -193,7 +195,8 @@ aileron_down_limit = 17 ; Aileron max deflection down angle (absolute value) (DE
 rudder_limit = 25 ; Rudder max deflection angle (absolute value) (DEGREES)
 rudder_trim_limit = 20 ; Rudder trim max deflection angle (absolute value) (DEGREES)
 elevator_trim_limit = 13.5 ; Elevator trim max angle (absolute value) (DEGREES)
-spoiler_limit = 35 ; Spoiler max deflection angle (absolute value) (DEGREES)
+spoiler_limit = 35 ; Spoiler max deflection angle when in the air (absolute value) (DEGREES)
+air_spoiler_limit = 35 ; Spoiler max deflection angle when on the ground (absolute value) (DEGREES)
```
Comment (sidnov) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg:199]: The descriptions look mixed up, ```spoiler_limit``` is "Spoiler max deflection angle, in degrees (absolute values only) when on the ground." and vice-versa

**Review by sidnov — APPROVED** [review]
Just a minor change in line 198, LGTM

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg (hunk 1 above)
Comment (donstim) [reply flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg:199]: Oops, good catch!  Thanks!

**Review by sidnov — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**